.github/workflows/create-lint-wf.yml

```diff
@@ -1,5 +1,11 @@
 name: Create a pipeline and run nf-core linting
-on: [push, pull_request]
+on:
+  push:
+    branches:
+      - dev
+  pull_request:
```

Comment:
You might consider only running CI on PRs to `master` or `dev`, as well.

Reply:
Ah, you're right, might as well on those, otherwise they won't get run when the release PR is made.

Reply:
Wait, this is already doing that. It runs on any PR, regardless of the target branch.

Reply:
Exactly, I usually only run CI on PRs targeting master/dev and not all PRs. Both can be useful, though.